sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -4136,6 +4136,13 @@ object GpuOverrides extends Logging {
     }
   }
 
+  /**
+   * To judge whether "spark.rapids.sql.explain" is explicitly set by the users.
+   */
+  private def isSQLExplainExplicitlySet(conf: RapidsConf) : Boolean = {
```

Comment:
sorry I may not have been clear enough, I meant add this to RapidsConf so that it could potentially be reused and you don't have to reach into rapidsConfMap.  I would make it generic so you pass can pass in any key.


Note, nit should be no space before : -> (conf: RapidsConf): Boolean

Reply:
@tgravescs Thanks for you patience, hope that this time I got what you mean. I have submitted a new commit to fix this.